2024-01-21 17:14:40.956 [info] Started local extension host with pid 691996.
2024-01-21 17:14:41.012 [error] Extension 'GitHub.vscode-pull-request-github CANNOT USE these API proposals 'tokenInformation, contribShareMenu, fileComments, contribCommentPeekContext, contribCommentThreadAdditionalMenu, codiconDecoration, diffCommand, contribCommentEditorActionsMenu, shareProvider, quickDiffProvider, readonlyMessage, treeViewMarkdownMessage'. You MUST start in extension development mode or use the --enable-proposed-api command line flag
2024-01-21 17:14:41.012 [error] Extension 'ms-python.python CANNOT USE these API proposals 'contribEditorContentMenu, quickPickSortByLabel, testObserver, quickPickItemTooltip, saveEditor, terminalDataWriteEvent, terminalExecuteCommandEvent'. You MUST start in extension development mode or use the --enable-proposed-api command line flag
2024-01-21 17:14:41.063 [error] [GitHub.vscode-pull-request-github]: comments/comment/editorActions is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribCommentEditorActionsMenu"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.vscode-pull-request-github
2024-01-21 17:14:41.064 [error] [GitHub.vscode-pull-request-github]: comments/commentThread/additionalActions is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribCommentThreadAdditionalMenu"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.vscode-pull-request-github
2024-01-21 17:14:41.064 [error] [GitHub.vscode-pull-request-github]: comments/commentThread/title/context is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribCommentPeekContext"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.vscode-pull-request-github
2024-01-21 17:14:41.065 [error] [GitHub.vscode-pull-request-github]: comments/commentThread/comment/context is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribCommentPeekContext"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.vscode-pull-request-github
2024-01-21 17:14:41.065 [error] [GitHub.vscode-pull-request-github]: editor/context/share is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribShareMenu"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.vscode-pull-request-github
2024-01-21 17:14:41.066 [error] [GitHub.vscode-pull-request-github]: file/share is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribShareMenu"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.vscode-pull-request-github
2024-01-21 17:14:41.066 [error] [GitHub.vscode-pull-request-github]: editor/title/context/share is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribShareMenu"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.vscode-pull-request-github
2024-01-21 17:14:41.067 [error] [GitHub.vscode-pull-request-github]: explorer/context/share is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribShareMenu"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.vscode-pull-request-github
2024-01-21 17:14:41.067 [error] [ms-python.python]: editor/content is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["contribEditorContentMenu"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api ms-python.python
2024-01-21 17:14:43.335 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:14:44.332 [info] [perf] Render performance baseline is 161ms
2024-01-21 17:14:49.874 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:14:51.568 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:14:52.582 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:14:54.166 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:14:55.364 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:14:56.897 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:03.960 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:06.938 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:08.130 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:10.220 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAExit (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:35)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16966)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:10.237 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAError (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1552:7629)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16827)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:10.865 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:11.207 [error] Cannot read properties of undefined (reading 'add'): TypeError: Cannot read properties of undefined (reading 'add')
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1552:2129)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.z (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1972)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2188)
    at vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1912:73743
2024-01-21 17:15:11.373 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:12.045 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAExit (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:35)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16966)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:12.051 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAError (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1552:7629)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16827)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:14.322 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:15.538 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:16.613 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:17.754 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:18.776 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:22.385 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:25.471 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:26.557 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:27.568 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:28.727 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:32.753 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:33.891 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAExit (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:35)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16966)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:33.892 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAError (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1552:7629)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16827)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:34.889 [error] Cannot read properties of undefined (reading 'add'): TypeError: Cannot read properties of undefined (reading 'add')
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1552:2129)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.z (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1972)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2188)
    at vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1912:73743
2024-01-21 17:15:36.365 [error] Cannot read properties of undefined (reading 'add'): TypeError: Cannot read properties of undefined (reading 'add')
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1552:2129)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.z (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1972)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2188)
    at vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1912:73743
2024-01-21 17:15:37.030 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:37.217 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAExit (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:35)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16966)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:37.219 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAError (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1552:7629)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16827)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:37.651 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:39.255 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:40.283 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:44.532 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:49.236 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:50.563 [error] No expression to evaluate
2024-01-21 17:15:50.567 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:52.155 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:54.303 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:55.366 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAExit (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:35)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16966)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:55.367 [error] Invalid debug adapter: Error: Invalid debug adapter
    at v.m (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1553:95)
    at v.$acceptDAError (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1552:7629)
    at m.S (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18889)
    at m.Q (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:18655)
    at m.M (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:17748)
    at m.L (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:16827)
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1561:15630)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2119)
    at s.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:616:14449)
    at x.onmessage (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1657:7802)
2024-01-21 17:15:56.399 [error] Cannot read properties of undefined (reading 'add'): TypeError: Cannot read properties of undefined (reading 'add')
    at o.value (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1552:2129)
    at u.y (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1902)
    at u.z (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:1972)
    at u.fire (vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:87:2188)
    at vscode-file://vscode-app/usr/lib/code/out/vs/workbench/workbench.desktop.main.js:1912:73743
2024-01-21 17:15:56.935 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-01-21 17:15:58.564 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339625
	at re (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2339919)
	at /home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/irene/.vscode-oss/extensions/ms-python.python-2023.22.1-universal/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
